package.json

```
@@ -5,7 +5,7 @@
     "main": "index.js",
     "scripts": {
         "test": "jest --verbose false",
-        "dev": "ts-node-dev -r tsconfig-paths/register --inspect --transpile-only --ignore-watch node_modules src/adapter/driver/api/index.ts ",
+        "dev": "ts-node-dev -r tsconfig-paths/register --inspect --transpile-only --ignore-watch node_modules src/api/index.ts ",
         "build": "tsc --project ./tsconfig.build.json && tsc-alias -p ./tsconfig.build.json",
         "start": "node build/adapter/driver/api/index.js",
```

pra conseguir rodar, eu precisei alterar aqui:

```suggestion
        "start": "node build/api/index.js",
```